v19/src/app/features/auth/login/login.component.html

```
@@ -0,0 +1,109 @@
+/* 
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)